oak/server/oak_node.cc

```diff
@@ -48,6 +49,15 @@ static void WriteMemory(wabt::interp::Environment* env, const uint32_t offset,
   std::copy(data.cbegin(), data.cend(), env->GetMemory(0)->data.begin() + offset);
 }
 
+static void WriteI32(wabt::interp::Environment* env, const uint32_t offset, const int32_t value) {
+  uint32_t v = static_cast<uint32_t>(value);
+  auto base = env->GetMemory(0)->data.begin() + offset;
+  *(base + 0) = v & 0xff;
```

Comment:
Just curious, would `base[0] = ...` work here?

Reply:
Yep, and that's probably clearer - done.